### DISCUSSION TITLE
How show lvl0 (category header in top)

### DISCUSSION THREAD

**Discussion by Zhengqbbb** [algolia/docsearch#2293] [Q&A]
In previous versions, I remember that there was a record showing lvl0 as the source category, placed above the results. 

However, in the recent version, it has disappeared. I wonder how to configure it to display again?

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/399f24ba-b9ef-4cc9-aaaa-62cf12bed82a">

<img width="1512" alt="image" src="https://github.com/user-attachments/assets/2af994d0-8e74-4206-aea2-826a5edc5b34">

**Comment by randombeeper** [comment]
Hello @Zhengqbbb, I wanted to thank you again for reporting this issue and we've discussed it under the Issues tab. @levimichael provided a workaround and we are still working with development to resolve this within the product.

I'm going to close this discussion as we are trying to keep interactions to our Discord at https://alg.li/discord and we'll be happy to help you there! This discussion area is not well monitored.
